src/main/java/com/yelp/nrtsearch/server/luceneserver/field/IdFieldDef.java

```diff
@@ -47,10 +47,10 @@ protected IdFieldDef(String name, Field requestField) {
    * @param requestField field properties to validate
    */
   protected void validateRequest(Field requestField) {
-    if (requestField.getMultiValued() || requestField.getTokenize()) {
+    if (requestField.getMultiValued()) {
```

Comment:
It looks like we're allowing tokenization on the ID field. Let's add that to the PR description as well.

Reply:
For `_ID` fields, the tokenization is hard coded to `false` https://github.com/aprudhomme/nrtsearch/blob/deprecate_tokenize_param/src/main/java/com/yelp/nrtsearch/server/luceneserver/field/IdFieldDef.java#L74 so the request values does not matter.